### ISSUE TITLE
check-declare error generated but lint still "OK"

### ISSUE THREAD

**Issue by gonewest818** [gonewest818/elisp-lint#26]
While linting anzu.el:
```
anzu.el:43:Warning (check-declare): said ‘migemo-forward’ was defined in
    unknown file: Malformed declaration
Indenting region... 
Indenting region...done
anzu.el OK
```

That warning was generated by check-declare but not caught by elisp-lint, hence the file is erroneously considered "ok".

**Comment by gonewest818** [comment]
This is an upstream issue ... `check-declare` doesn't report malformed declarations in its output.

**Comment by gonewest818** [comment]
http://debbugs.gnu.org/cgi/bugreport.cgi?bug=40188

**Comment by gonewest818** [comment]
Noting here that the above Emacs bug report was closed. The maintainers are appacrently satisfied with the behavior.
